aws/monitoring.tf

```diff
@@ -13,7 +13,7 @@ resource "aws_sns_topic_subscription" "email" {
     compact(
       concat(
         var.monitoring.email_alerts_datagrok ?
-        ["monitoring@datagrok.ai", "spodolskaya@datagrok.ai", "dnahovskyi@datagrok.ai"] :
+        ["monitoring@datagrok.ai", "spodolskaya@datagrok.ai", "vhlushchenko@datagrok.ai"] :
```

Comment:
Remove monitoring@datagrok.ai, it is useless, unfortunately, due to AWS email spam

Reply:
@VHlushchen

Reply:
Removed